docs/hardware/alarms.md

```
@@ -0,0 +1,204 @@
+    tt: Math.floor(Date.now() / 1000),
+  }
+  return Buffer.from(encoder.encode(payload)).toString('hex')
+}
+```
+
+### Worked example
+
+`config = { vibrationIntensity: 60, vibrationPattern: 'rise', duration: 15 }` at
+unix time `1778486725` encodes to:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)